docs/user/user-defined-redirects.rst

```diff
@@ -84,6 +84,20 @@ For example::
 You can choose which is the :term:`default version` for Read the Docs to display.
 This usually corresponds to the most recent official release from your project.
 
+Root language redirect at ``/<lang>/``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A link to the root language of your documentation (`<slug>.readthedocs.io/en/`)
+will redirect to the  :term:`default version` of that translation,
+as set in your project settings.
+
+This works for readthedocs.io (|org_brand|) and :doc:`custom domains </custom-domains>`.
+For |com_brand| this can be enabled by contacting :doc:`support </support>`.
```

Comment:
Is the new Proxito implementation enabled for all .org projects?

```suggestion
.. TODO: Remove this once the feature is default on .com

This redirect is currently only active on |org_brand| (``<slug>.readthedocs.io`` and :doc:`custom domains </custom-domains>`).

Root language redirects on |com_brand| can be enabled by contacting :doc:`support </support>`.
```

Reply:
> Is the new Proxito implementation enabled for all .org projects?

yeah, in .com we have our projects only, we could start testing more widely on .com, but we need to make sure to not include projects with custom urlconfs, since that feature isn't supported yet.